### PR TITLE
Add Conone 65

### PR DESCRIPTION
**Pull request by evyd13**
## Description

Adds the Conone 65 keyboard to QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kindakeyboards/conone65/config.h [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x1776
```
Comment (drashna) [inline keyboards/kindakeyboards/conone65/config.h:23]: This vendor ID is already in use.  Please change.

https://www.the-sz.com/products/usbid/index.php?v=0x1776&p=&n=

File: keyboards/kindakeyboards/conone65/config.h [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+#define PRODUCT_ID   0x6aab
```
Comment (drashna) [inline keyboards/kindakeyboards/conone65/config.h:24]:
```suggestion
#define PRODUCT_ID   0x6AAB
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/kindakeyboards/conone65/readme.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Conone 65
+
+*The Conone 65 is a custom keyboard designed and machined in Belgium.*
```
Comment (fauxpark) [inline keyboards/kindakeyboards/conone65/readme.md:3]:
```suggestion
The Conone 65 is a custom keyboard designed and machined in Belgium.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks for your contribution!

**Pull request merged**